app/src/app/components/contents/explorer/utils/getTimeDifferenceToDisplay.ts

```diff
@@ -0,0 +1,15 @@
+export const getTimeDifferenceToDisplay = (unixTime: number): { unit: string; value: number } => {
```

Comment:
This seems to be doing a lot of the same work as `getElapsedTime`, could they be combined?
https://github.com/mrc-ide/packit/blob/aefaac76b342b654e6f42774ef5eae617c7b6cc5/app/src/helpers.ts#L17

Reply:
yup il leave a todo and when i do the packet pages next il sort as part of that